typescript/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  root: true,
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: { project: "./tsconfig.json", tsconfigRootDir: __dirname },
```

Comment:
I am unfamiliar with the nuances of the eslint, is the tsconfigRootDir required as a parserOption? I noticed it wasn't set in the LLama extension

Reply:
It's not required, but seems needed in this case to have it found in the __tests__. It's not set in the llama extension since there's no __tests__ dir where the issue would be seen (or any directory excluded from compilation in tsconfig)